packages/microcosm/src/microcosm.js

```diff
@@ -234,10 +233,10 @@ class Microcosm extends Emitter implements Domain {
    */
   append(command: *, status?: Status): Action {
     if (command in this.actions) {
-      return this.history.append(this.actions[command], status)
+      return this.history.append(this.actions[command], status, this)
```

Comment:
could this be used to instantiate instances of microcosm trees that don't process actions outside of their microcosm branch?

Reply:
Good question. I don't think so, but it might be a first step towards it.

We need to do some planning for making a microcosm that doesn't listen to _certain_ actions but not others. Right now every Presenter forks it's associated repo. I'm afraid that it would be hard to tell why an action isn't getting responded to.

This might need to happen inside of a domain. I wonder if there's a concept that we're missing like a public/private domain. Private domains could be setup such that they didn't respond to actions on their parents or siblings. 

It's a good thing for us to sort out for 13.x